EMIP/1-1000/EMIP00339.xml

```diff
@@ -37,8 +38,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </msIdentifier>
                <msContents>
                   <summary>Four Gospels of Empress Zäwditu; </summary>
-                  <msItem xml:id="ms_i1">
+                  <msItem xml:id="ms_i1"><locus from="1r" to="51r"/><title type="complete" ref="LIT1558Matthew"/>
+                  <textLang mainLang="gez">Gə‘əz</textLang>
+                  <incipit xml:lang="gez">ብሥራተ: አብ: ትሩፍ: ማቴዎስ: ሐዋርያ: ፩እም፲ወ፪ቱ: ሐዋርያት: በረከተ: አምላኩ: ውድስት። ወሀብተ: ጸሎቱ: ውክፍት: ተሀሉ: ምስለ: ንግሥትነ: አስካለ: ማርያም: ወምስለ: ፍቁራ: ወራሴ: መንግሥት: ኃይለ: ሥላሴ: ለዓለመ: ዓለም: አሜን።</incipit>
+                </msItem>
+                <msItem xml:id="ms_i1.1"><locus from="51r" to="51r"/>
+                <textLang mainLang="gez">Gə‘əz</textLang>
+                <title>Conclusion of the Gospel of Matthew</title>
+                <incipit xml:lang="gez">መልዓ; ብሥራተ: ማቴዎስ: ሐዋርያ: ወኮነ: ጸሐፎ: በምድረ: ፍልስጥዔም: በአስተሐምሞ: ቅዱስ: እብራይስጣዊ: እምድኅረ: ዕርገቱ: ለእግዚእነ: ኢየሱስ ክርስቶስ: በአጶርግዮ: ለኀበ: ሰማይ: በሰማኒቱ: ዓመታት: በቀዳሚሁ: አም: ዘእምቀላውዴዎስ: ቄሣር: ንጉሠ: ሮም: ወስብሐት: ለእግዚአብሔር: ለዓለመ: ዓለም። አሜን።</incipit>
```

Comment:
Translation should be added  through 
                  <foreign xml:lang="en">
                  ...... </foreign>
                  </incipit>
 See https://betamasaheft.eu/Guidelines/?q=translation&id=manuscriptContents